src/models.rs

```diff
@@ -90,7 +90,7 @@ mod tests {
     impl Model for OnlyOneKindModel {
         type ColorFormat = Rgb565;
 
-        const FRAMEBUFFER_SIZE: (u16, u16) = (0, 0);
+        const FRAMEBUFFER_SIZE: (u16, u16) = (16, 16);
```

Comment:
This was done since it actually fails with 0, 0 now (as it should).

NOTE: It'd be great if we cound enforce > 0 on this constant but I guess it's not compile time possible?